jobs/e2e-test.groovy

```diff
@@ -183,6 +183,7 @@ def runLambdaTest() {
                             "-p=${params.NUM_WORKER_MACHINES}",
                             "--sync=true",
                             "--bt='jenkins,${e2eEnv}'",
+                            "--tags='jenkins,${e2eEnv}'",
```

Comment:
**note:** We have to duplicate the tags here (bt and tags) due to the way LambdaTest exposes its CLI :|
- `bt` stands for `buld-tags`, which are tags associated to a build (or a session that runs multiple test specs).
- `tags` are the tags associated to each test spec.

Reply:
Thanks for this comment.  It would be very useful to include it here since to the untrained eye the duplication does not seem obviously intentional.  For the future tho.